<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>administratorUser</class>
 <widget class="QDialog" name="administratorUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员页面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget QToolButton{border:none;background-color:rgb(84,92,100);min-height:55px;font-size:14px;font-family:&quot;黑体&quot;;color:white;}
QWidget#widget QToolButton:checked{border:none;background-color:rgb(67,74,80);}
QWidget#widget {background-color:rgb(84,92,100);}
QDialog{background-color:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <widget class="QToolButton" name="btn_stupa">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>学生奖惩</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QToolButton" name="btn_courseSet">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>开设课程</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="11" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="0">
       <widget class="QToolButton" name="btn_administrator">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>管理员管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QToolButton" name="btn_tea">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>教师管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="btn_stu">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>学生管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="btn_takes">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>学生选课</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QToolButton" name="btn_course">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>课程管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QToolButton" name="btn_college">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>学院管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QToolButton" name="btn_score">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string>成绩管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QPushButton[name=&quot;btn&quot;]{font-size:14px;min-height:28px;padding:4px;min-width:85px}
QLineEdit[name=&quot;le&quot;]{border-radius:4px;min-height:25px;border:1px solid gray;min-width:280px;max-width:280px;}
QLabel[name=&quot;lb&quot;]{font:14px '黑体';}
QLabel[name=&quot;title&quot;]{font:18px '宋体';}
QWidget[name=&quot;bg&quot;]{background-color:white;}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
